components/raftstore-v2/tests/integrations/test_read.rs

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::assert_matches::assert_matches;
+
+use futures::executor::block_on;
+use kvproto::{
+    kvrpcpb::Context,
+    raft_cmdpb::{CmdType, GetRequest, RaftCmdRequest, ReadIndexRequest, Request, StatusCmdType},
+};
+use raftstore::store::util::new_peer;
+use tikv_util::{
+    codec::number::NumberEncoder,
+    thread_group::{set_properties, GroupProperties},
+};
+use txn_types::WriteBatchFlags;
+
+#[test]
+fn test_read_index() {
+    set_properties(Some(GroupProperties::default()));
```

Comment:
Properties is used for detect shutdown when clearing callbacks. In v2, we can have other way to detect such failure. I suggest not to include it.

Reply:
Without this line, it will panic.